.github/workflows/pypi-package.yml

```
@@ -0,0 +1,52 @@
+---
+name: Build and upload package
+
+on:
+  push:
+    branches: [main]
+    tags: ["*"]
```

Looks like we also need:

```suggestion
  push:
    branches: [main]
    tags: ["*"]
  pull_request:
    branches: [ main ]
```